deploy/helm/values.yaml

```diff
@@ -284,9 +284,21 @@ otel:
 
   # Configuration for Events collection
   events:
-    # Define whether events will be collected and sent
+    # If false none of the events will be collected
     enabled: true
 
+    # If true, the k8s events will be collected
+    k8s_events: true
+
+    # Configuration that allows to watch for changes in k8s resources and send full manifests to SWO upon change
+    manifests:
```

Comment:
I get why you put this in the `events` section. But logically, there is no relation between collection of manifests and collection of events. They just happen to run in the same container. The current implementation also means that all filtering and k8s instrumentation settings must apply to both events and manifests equally.

Reply:
Yes I set to pipeline to run in events collector and introduced manifests as second type of events. I turned k8s event collector to just event collector which scrape two types of events. It made sense to me as it is kind of event that we will send to another event namespace. Therefore `events.enabled` is turning on/off events collector, `events.k8s_events` is turning on/off k8s events collection and `events.manifests.enabled` is turning on/off manifests collection. Also since it all runs in the same deployment it logically share batching, queue configuration, timeouts, resource configuration etc. Having shared filter is not ideal though. It would be probably better to introduce new section for shared stuff, like `event_collector`, move there resource configuration, telemetry, batching, timeout etc.and then having manifests separate. What do you suggest?

Reply:
After discussion I moved `manifests` to its own section.